package.json

```diff
@@ -9,6 +9,12 @@
   "files": [
     "dist/*.ts",
     "dist/*.js",
+    "dist/gen/*.ts",
```

Comment:
https://docs.npmjs.com/files/package.json#files

Do you think we could do 

`"dist/**/*.js"`
`"dist/**/*.ts"`

Wondering if this would be better then we don't need to remember as the folders change?

Reply:
Would that pull in the examples?  e.g., `dist/exec/`

Use the below instead?
```
"dist/gen/**.js",
"dist/gen/**.ts"
```

Reply:
@silasbw Why would you want to explicitly only include gen?

Is there something different between what I suggested above here? 

```
"dist/**/*.js"
"dist/**/*.ts"
```